apps/humanatlas.io/project.json

```diff
@@ -23,10 +23,16 @@
         "assets": [
           "apps/humanatlas.io/src/favicon.png",
           "apps/humanatlas.io/src/assets",
+          "apps/humanatlas.io/src/_redirects",
```

Comment:
Revert changes in this file (what you're doing in github workflows is the better solution)